docs/build/avalanchego-apis/admin.md

```diff
@@ -203,6 +203,47 @@ curl -X POST --data '{
 }
 ```
 
+### admin.loadVMs
+
+Dynamically loads any virtual machines installed on the node as plugins.
```

Comment:
Can we put more information like where these plugin binaries should be put, in which form etc? We already have this information in https://docs.avax.network/build/tutorials/platform/subnets/create-a-virtual-machine-vm#executable-binary we can try to refer there but probably best to have a separate section (titled with AvalancheGo plugins etc) for the part:
> AvalancheGo searches for and registers plugins under [buildDir]/plugins/. You need to put built VM binary under this path. The [buildDir] defaults to the path of executed AvalancheGo binary. See [here](https://docs.avax.network/build/references/avalanchego-config-flags#build-directory) for more information. Executable names must be either a full VM ID (encoded in CB58), or must be a VM alias as explained below.

alternatively we can move this part under the https://docs.avax.network/build/references/avalanchego-config-flags/#build-directory and refer this section in other two pages (in VM page and in Admin API page)

Reply:
@joshua-kim can you look over @ceyonur 's comment above and resolve it?

Reply:
Added a sub-section under the VM installation tutorial

Reply:
👍